docs/sql-ref-datetime-pattern.md

```diff
@@ -63,7 +63,10 @@ Spark uses pattern letters in the following table for date and timestamp parsing
 
 The count of pattern letters determines the format.
 
-- Text: The text style is determined based on the number of pattern letters used. Less than 4 pattern letters will use the short form. Exactly 4 pattern letters will use the full form. Exactly 5 pattern letters will use the narrow form. 5 or more letters will fail.
+- Text: The text style is determined based on the number of pattern letters used. Less than 4 pattern letters will use the short form. Exactly 4 pattern letters will use the full form. Exactly 5 pattern letters will use the narrow form. 5 or more letters will fail. More details for the text style:
```

Comment:
How about
```
... will use the short form (typically an abbreviation, for example, day-of-week Monday might output "Mon"). ...
```

BTW we should remove `Exactly 5 pattern letters will use the narrow form`